tests/actions/buildiso/append_line_test.py

```diff
@@ -71,6 +71,26 @@ def test_generate_profile(request, cobbler_api, create_distro, create_profile):
         % (request.node.originalname, request.node.originalname)
     )
 
+def test_generate_profile_install(request, cobbler_api, create_distro, create_profile):
+    # Arrange
+    test_distro = create_distro()
+    test_distro.kernel_options = "install=http://192.168.40.1:80/cblr/links/%s" % request.node.originalname
```

Comment:
As much as this is correct, the SUSE internal tests require us to add a little something here:

```suggestion
    test_distro.kernel_options = "install=http://192.168.40.1:80/cblr/links/%s" % (
            request.node.originalname
            if request.node.originalname
            else request.node.name
        )
```

This also applies to other usages of this fixture. The reason is that in the OBS for Leap we have an outdated pytest version possibly that is not compatible with this fixture.

Reply:
See here for example usage: https://github.com/cobbler/cobbler/blob/3f516747ca168dcff8e8fe3c26f69c906e65b752/tests/conftest.py#L105

Reply:
Hmm.. I am using same as in other tests in this file. The rest of the tests there uses `request.node.originalname` in their asserts.

Reply:
Then the other tests in this file are wrong. The RPM based version of pytest is very outdated and thus doesn't have the new style name.

Reply:
Ok, updated to use either `.originalname` or if unavailable to use `.name`.